src/components/Form/UploadDocumentForm.jsx

```diff
@@ -144,7 +154,7 @@ const UploadDocumentForm = () => {
               color="primary"
               id="upload-doctype"
               name="uploadDoctype"
-              onChange={handleFileChange}
+              onChange={(e) => setFile(e.target.files[0])}
               required
             >
               Choose file
```

Comment:
It was missed in an earlier PR, but I found the issue now. In Line 161, the `accept` is missing a comma between .rtf and .gif. It'll miss those files as acceptable file extensions.

Reply:
fixed